### DISCUSSION TITLE
New Bindings PR

### DISCUSSION THREAD

**Discussion by seanaye** [urql-graphql/urql#2492] [Q&A]
Would you accept a PR for solid js bindings? Or is one being worked on already?

**Comment by JoviDeCroock** [comment]
I think @colinrobertbrooks was working on that

**Reply by colinrobertbrooks** [reply]
@lukejacksonn & I have been working with this. We're down to subscriptions, then tests.

**Reply by seanaye** [reply]
great, let me know if you need any help, happy to contribute

**Reply by colinrobertbrooks** [reply]
See https://github.com/FormidableLabs/urql/pull/2527 cc @gksander
